realm/realm-library/src/testUtils/java/io/realm/TestHelper.java

```diff
@@ -76,7 +78,7 @@
     public static final int STANDARD_WAIT_SECS = 200;
 
     private static final Charset UTF_8 = Charset.forName("UTF-8");
-    private static final Random RANDOM = new Random();
+    private static final SecureRandom RANDOM = new SecureRandom();
```

Comment:
Is SecureRandom more random than Random?

Reply:
According to the documentation it's more secure when used in the context of cryptography, which is the case with `encryptionKey` parameter of the RealmConfiguration. This is more to set the example for developers